INSTALL.md

```diff
@@ -17,8 +35,6 @@ None.
 
 > Android is not supported using the `install.sh` script.
 
-Choose steps matching your plugin manager.
```

Comment:
Why this getting removed?

Reply:
I feel that it's a bit redundant with L27, L38, and L40. I'd be happy to revert the change to this line if you feel like it adds contextual value.